cloudbuild/e2e-test.yaml

```diff
@@ -45,8 +45,8 @@ steps:
         curl -sSL https://install.python-poetry.org | python3 - && \
         export PATH="/builder/home/.local/bin:$$PATH" && \
         make install && \
-        make e2e-tests pipeline=training && \
-        make e2e-tests pipeline=prediction
+        make run pipeline=training build=false wait=true && \
+        make run pipeline=prediction build=false wait=true
```

Comment:
@browningjp-datatonic why don't we build containers when running e2e tests?

Reply:
The containers are built in the previous pipeline step. We pass `build=false` here because `make build` would create a separate Cloud Build job (which fails in this case, because it relies on the `gcloud` command that is not available in the CI container image)